### PR TITLE
second commit

### PR DESCRIPTION
**Pull request by Perpee**
please check my work @promise-J and @ChidiPatrick

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
\ No newline at end of file
+##Campeonas '22_Pharmacy school
+
+##How to run/test this project
+
+##ensure you have git installed properly on your machine
+
+##ensure you have node js installed in your machine
+
+##open your terminal and run "git clone "
```
Comment (Pieta9221) [inline README.md:9]: Kindly add the repo link

File: html/index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            school,
+            <a href="school.html" title="COOU">
+                Chukwuemeka Odumegwu Ojukwu University
+            </a>, Anambra state happens to be one of such higher institutions and
+            has its Faculty of Pharmaceutical Sciences situated at the Igbariam
+            campus.
+            <br />
+            We have six
+            <a
+                href="department.html"
```
Comment (Pieta9221) [inline html/index.html]: Use the 'a' tag and the "href" attribute to create the anchor link, and the <a> tag with the "name" attribute to create the anchor target.
This wasn't implemented

File: html/index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+                <a href="#quality">Qualities of a good pharmacist</a>
+            </nav>
+        </header>
+        <p>
+            Pharmacy is a course of study which produces professionals in the
+            field of drug manufacturing, production, componding, distribution
+            and use. It is act, practice or profession of preparing drugs
+            according to Doctor's prescription. The pharmacy profession is
+            cordinated by the body
+            <a href="pcn.html" title="pcn">Pharmacists Council of Nigeria (PCN)</a>.
```
Comment (Pieta9221) [inline html/index.html]: Within the paragraph, create at least three different links to external websites that provide additional information on your chosen topic.
The links are not leading to external websites

**Review by Pieta9221 — COMMENTED** [review]
:+1: :+1: :+1:
Kindly effect the changes before merging

File: html/index.html (hunk 2 above)
Comment (Perpee) [reply html/index.html]: Thanks for the review. Do you mean something like this? name="Departments in my school of pharmacy"

**Pull request merged**